## id false为绝对位置起飞，true为相对位置起飞，x y z绝对起飞位置，h相对起飞高度
take_off:
  id: false
  x: 0.0
  y: 0.0
  z: 8.0
  h: 5.0

## 集结位置
assemble:
  x: 20.0
  y: 20.0
  z: 15.0
  
## 返航位置
home:
  x: 0.0
  y: 0.0
  z: 15.0

## 降落位置
landing:
  x: 1.0
  y: 2.0
  vz: -0.5